### PR TITLE
Add inspiration to module list, fix stacking from repeat gain events

### PR DESCRIPTION
**Pull request by ackwell**
## Pull request type

- [ ] This is new functionality or an addition to existing functionality
- [x] This is a bugfix to existing functionality
- [ ] This is a patch bump

## Pull request details

- [ ] This is in response to a GitHub issue: *[Link to issue]*
- [x] This is in response to a discussion or thread on Discord:
    - https://discord.com/channels/277897135515762698/1192242035516788797/1271742792435109929
    - https://discord.com/channels/277897135515762698/1192242035516788797/1271821732705210441
- [ ] The goal of this PR is detailed below:

## Testing / Validation

- [ ] The changes being implemented with this bugfix include comments that contain example log(s) to test with
  - [ ] I have submitted the example log(s) to the xivanalysis static on FFLogs for record keeping
- [x] I used the log(s) listed below to develop and test this bugfix:
	- inspiration: https://www.fflogs.com/reports/2q93AYbnKCcRGjhZ#fight=17&source=7
	- end of fight RD: https://www.fflogs.com/reports/G3ZxA21LKPX8c9M6#fight=10&source=26

## Job Maintenance
hi it me

### CODE REVIEW

**Comment by ackwell** [comment]
i should add a changelog hey

File: src/data/CONTRIBUTORS.ts [hunk 1]
```diff
@@ -31,6 +31,11 @@ const CONTRIBUTORS = ensureRecord<Contributor>()({
 			JOBS.BLACK_MAGE, JOBS.DANCER, JOBS.SAGE, JOBS.PICTOMANCER,
 		],
 	},
+	ACKWELL: {
+		name: 'ackwell',
+		avatar: require('./avatar/ackwell.png'),
+		jobs: [JOBS.PICTOMANCER],
```
Comment (ackwell) [inline src/data/CONTRIBUTORS.ts:37]: oh how the turns table

**Comment by ackwell** [comment]
Also fixed RD's recast adjustment having no end time, leading to all RDs after the first buffed one being a short cast.

**Review by AkairyuGestalter — APPROVED** [review]
🤦 lgtm though!

**Pull request merged**